app/src/main/java/com/github/se/cyrcle/ui/gambling/GamblingScreen.kt

```diff
@@ -0,0 +1,306 @@
+package com.github.se.cyrcle.ui.gambling
+
+import android.util.Log
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.geometry.Size
+import androidx.compose.ui.graphics.*
+import androidx.compose.ui.graphics.drawscope.rotate
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.contentDescription
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.drawText
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.rememberTextMeasurer
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.github.se.cyrcle.R
+import kotlin.math.*
+import kotlinx.coroutines.*
+
+@Composable
+fun WheelView(
+    modifier: Modifier = Modifier,
+    onSegmentLanded: ((String) -> Unit)? = null
+): () -> Unit {
+  data class Segment(val name: String, val color: Color, val probability: Float)
+
+  val segments = remember {
+    listOf(
+        Segment("Nothing", Color.Gray, 50f),
```

Comment:
These strings could also be placed in `string.xml`

Reply:
As I said in another comment, I dont think these strings will stay written here on the segments, but in the meantime I will indeed add them to string.xml.